File: go/string/1446. 连续字符.go
package string

func maxPower(s string) int {
	res := 1
	for i := 1; i < len(s); i++ {
		tmp := 1
		for s[i] == s[i-1] {
			tmp++
			i++
		}
		if tmp > res {
			res = tmp
		}
	}
	return res
}

File: python/字符串/1446. 连续字符.py
class Solution:
    def maxPower(self, s: str) -> int:
        res = 1
        i = 1
        while i<len(s):
            tmp = 1
            while i<len(s) and s[i]==s[i-1]:
                tmp+=1
                i+=1
            if tmp>res:
                res = tmp
            i+=1
        return res
